Content.Client/GameObjects/Components/HUD/Inventory/HumanInventoryInterfaceController.cs

```diff
@@ -67,27 +72,40 @@ void AddButton(out ItemSlotButton variable, Slots slot, string textureName)
             AddButton(out _hudButtonPocket1, Slots.POCKET1, "pocket");
             AddButton(out _hudButtonPocket2, Slots.POCKET2, "pocket");
             AddButton(out _hudButtonBack, Slots.BACKPACK, "back");
+            AddButton(out _hudButtonOClothing, Slots.OUTERCLOTHING, "suit_storage");
```

Comment:
Why did you use suit_storage here. We have suit.

Reply:
Fixed.